src/qibo_cloud_backends/aws_client.py

```diff
@@ -0,0 +1,232 @@
+        transpiled_circuit = transpile(circuit_qiskit,
+                                       basis_gates = self.native_gates,
+                                       optimization_level = self.optimization_level,
+                                       coupling_map = self.custom_coupling_map)
+        transpiled_circuit_qasm = qasm2.dumps(transpiled_circuit) # Convert back to qasm.
+        transpiled_circuit_qasm = self.remove_qelib1_inc(transpiled_circuit_qasm)
+        transpiled_circuit_qasm = self.qasm_convert_gates(transpiled_circuit_qasm)
+        braket_circuit = BraketCircuit.from_ir(transpiled_circuit_qasm)
+        return braket_circuit
+
+    def execute_circuit(self,
+                        circuit,
+                        nshots=1000,
+                        verbatim_circuit=False,
+                        **kwargs):
+        """Executes a circuit (Qibo or Qibo-transpiled-to-Braket) on an AWS Braket device. The device defaults to the LocalSimulator().
+           This function allows the user to opt for verbatim_circuit on the device by indicating `verbatim_circuit=True`.
+            
+
+        Args:
+            circuit (quantum circuit): Either qibo.models.Circuit or braket.circuits.Circuit to execute on the Braket device.
+            nshots (int): Total number of shots.
+            verbatim_circuit (bool): If `True`, wrap the Braket circuit in a verbatim box to run it on the QPU without any transpilation.
+                Defaults to `False`.
+            # kwargs (dict): Additional keyword arguments passed to the AWS backends' `run()` method. At the moment, I dont know of any.
+
+        Returns:
+            Measurement outcomes (qibo.measurement.MeasurementOutcomes): The outcome of the circuit execution.
+        """
+        
+        if kwargs:
+            raise_error(
+                NotImplementedError,
+                "The use of additional arguments to `run()` is not supported yet.",
+            )
+
+        circuit_type = self.get_circuit_type(circuit)
+
+        if circuit_type == "Qibo":
+            # If Qibo circuit, run these to convert to Braket circuit.
+            measurements = circuit.measurements
+            if not measurements:
+                raise_error(RuntimeError, "No measurement found in the provided circuit.")
+            nqubits = circuit.nqubits
+            circuit_qasm = circuit.to_qasm()
+            circuit_qasm = self.remove_qelib1_inc(circuit_qasm)
+            circuit_qasm = self.qasm_convert_gates(circuit_qasm)
+            braket_circuit = BraketCircuit.from_ir(circuit_qasm)
+
+        elif circuit_type == "Braket":
+            # Create measurements = qibo.circuit.measurements
+            qibo_circuit = QiboCircuit(len(circuit.qubits))
+            for i in range(0,len(circuit.qubits)):
+                qibo_circuit.add(gates.M(i))
+            measurements = qibo_circuit.measurements
+            braket_circuit = circuit
```

Comment:
No need to support Braket circuits directly, as they would be run directly from the Braket SDK.